readme.md

```diff
@@ -54,13 +54,10 @@ Compilation
 Follow the instructions at the [megasource][megasource] repository page.
 
 ### *nix
-Run `platform/unix/automagic` from the repository root, then run ./configure and make.
+First pick a directory to store generated files to, say `build/` and then:
```

Comment:
```suggestion
Because in-tree builds are not allowed, you have to create Makefiles in a build folder.
In this example we will use the `build/` folder:
```